tests/TestCase/Routing/RouteBuilderTest.php

```diff
@@ -775,13 +775,13 @@ public function testResourcesOnlyArray(): void
 
         $result = $this->collection->routes();
         $this->assertCount(2, $result);
-        $this->assertSame('/articles', $result[1]->template);
-        $this->assertSame('index', $result[1]->defaults['action']);
-        $this->assertSame('GET', $result[1]->defaults['_method']);
+        $this->assertSame('/articles', $result[0]->template);
+        $this->assertSame('index', $result[0]->defaults['action']);
+        $this->assertSame('GET', $result[0]->defaults['_method']);
 
-        $this->assertSame('/articles/{id}', $result[0]->template);
-        $this->assertSame('delete', $result[0]->defaults['action']);
-        $this->assertSame('DELETE', $result[0]->defaults['_method']);
+        $this->assertSame('/articles/{id}', $result[1]->template);
+        $this->assertSame('delete', $result[1]->defaults['action']);
+        $this->assertSame('DELETE', $result[1]->defaults['_method']);
```

Comment:
Does this change in behavior concern you? I think its probably ok. Userland code relying on the previous ordering was likely brittle anyways.

Reply:
I don't think anyone should be relying on the ordering of the routes in the routes list. Ideally we should avoid it in the tests too but can't think of a way right now to do so.

Reply:
As long as the order of cloaking doesn't change, it should be fine.